docs/version-history/lts-versions-23-0.rst

```diff
@@ -66,10 +66,16 @@ Bugs Fixed:
   *condor_schedd* that's on the same private network.
   :jira:`2115`
 
+- Fixed a but that could cause jobs be created incorrectly when a submit
+  filed used :macro:`initialdir` and :macro:`max_idle` or :macro:`max_materialize`
+  in the same submit file.
+  :jira:`2092`
+
```

Comment:
```suggestion
- Fixed a but that could cause jobs to be created incorrectly when a submit
  filed used :subcom:`initialdir` and :subcom:`max_idle` or :subcom:`max_materialize`
  in the same submit file.
  :jira:`2092`

```

Reply:
does that work in 23.0 stable?

Reply:
Totally forgot this was stable... no subcom doesn't, but using macro is also wrong because macro is used for configuration options

Reply:
pushed your rephrase with a removal of :macro: